get_data/src/es_utils.py

```diff
@@ -19,7 +21,14 @@ def get_es_session(host_ip, port):
         user = ""
         pwd = ""
     try:
-        es = elasticsearch.Elasticsearch([{"host": host_ip, "port": port, "timeout": 10}], http_auth=(user, pwd))
+        # ssl_context = ssl.create_default_context(cafile="conf/ca.crt")
+        # ssl_context = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+        # ssl_context.verify_mode = ssl.CERT_OPTIONAL
+        # ssl_context.check_hostname = False
+        # ssl_context.load_verify_locations(cafile="conf/ca.crt")
+        # es = elasticsearch.Elasticsearch([{"host": host_ip, "port": port, "timeout": 11}], http_auth=(user, pwd), scheme="https", port=443, ssl_context=ssl_context)
```

Comment:
Do you need all the commented lines ?

Reply:
oui c'est sale sry, j'ai pas encore réussi a me connecter en SSL avec le certificat en local (ligne en comment), pour le moment j'ignore le certificat ce qui est pas top mais fonctionnel et comme on a besoin de push sur la BDD je push comme ca pour le moment...